ui/apps/platform/src/Containers/Collections/CollectionForm.tsx

```diff
@@ -141,6 +148,19 @@ function CollectionForm({
         }),
     });
 
+    const errors = {
+        ...formikErrors,
```

Comment:
A thought which comes to mind, not necessarily better: what is pro and con if a pure helper function at module scope has `formikErrors` and `saveError` as arguments? Early return if `saveError` is undefined, and encapsulate conditional logic for a possibly growing list of errors?

Reply:
I think the main con is the additional memory allocation right now, which is probably negligible. Since it is just the one error being appended I felt like it was easier to follow to do this inline during render instead of calling out to a helper function. Definitely agree that if the list of errors that we want to attach to Formik validation grows that this would be cleaner to pull out into a separate function though.